### PR TITLE
Project Style

### PR DESCRIPTION
**Pull request by Obimbo07**
This pull request introduces the project's full styles and design.

### CODE REVIEW

File: src/components/BookForm.js [hunk 1]
```diff
@@ -24,18 +24,19 @@ const BookForm = () => {
...
+          <option value="Science">Science</option>
+          <option value="Mystery">Mystery</option>
+          <option value="Romance">Romance</option>
+          <option value="Action">Action</option>
+          <option value="Technology">Technology</option>
         </select>
-        <button type="submit">Add Book</button>
+        <button className="addbtn" type="submit">Add Book</button>
       </form>
     </div>
```
Comment (shahadat3669) [inline src/components/BookForm.js]: 👍 Great job on getting the project to work functionally! However, for a more polished look that aligns with the design, consider making the following adjustment:

- **Alignment**: In the design, the elements in this section should align along the vertical axis with the book card. To achieve this, you may need to work on the alignment and positioning of the elements in your code. Ensuring that everything lines up neatly will give your project a more professional and visually appealing appearance.

| Your | Design |
| --- | --- |
|  image| image|

By updating the alignment, your project will be in better harmony with the design, providing a more cohesive and polished user experience. Keep up the good work! 👍📐🎨

File: src/components/BookForm.js [hunk 2]
```diff
@@ -24,18 +24,19 @@ const BookForm = () => {
   };
   return (
     <div className="form-section">
-      <form onSubmit={handleFormSubmit}>
+      <h2>ADD BOOK NEW</h2>
```
Comment (shahadat3669) [inline src/components/BookForm.js]: 👍 Good work on adding a title for the contact form section! However, to ensure your project aligns perfectly with the design, there are a couple of things to consider:

1. **Styling**: The styling of the title, such as font family, font weight, line height, font size, letter spacing, and color, should match the design. Make sure to update your CSS code to reflect these design specifications for the title.

2. **Horizontal Line**: In the design, there is a horizontal line that appears before this title. To achieve the desired look, you should add this horizontal line exactly as depicted in the design. This subtle addition will contribute to the overall visual coherence of your layout.

| Your | Design |
| --- | --- |
|  image| image|

By making these adjustments, you'll ensure that your project not only includes the required elements but also matches the design specifications. Keep up the great work! 👍🎨🖋️

File: src/components/BookForm.js [hunk 3]
```diff
@@ -24,18 +24,19 @@ const BookForm = () => {
...
-          <option value="technology">Technology</option>
+          <option value="Fantasy">Fantasy</option>
+          <option value="Science">Science</option>
+          <option value="Mystery">Mystery</option>
+          <option value="Romance">Romance</option>
+          <option value="Action">Action</option>
+          <option value="Technology">Technology</option>
         </select>
-        <button type="submit">Add Book</button>
+        <button className="addbtn" type="submit">Add Book</button>
```
Comment (shahadat3669) [inline src/components/BookForm.js:34]: 👍 Good progress on adding the button for the form, and the color is correct. To bring it even closer to the design, here are a few suggestions:

- **Button Styling**: Update the button to match the design more closely. You can start by adjusting the border radius, padding, and font styles, such as font family and font size. Ensure that these styles align with the design guidelines. 

| Your | Design |
| --- | --- |
|  image| image|
By following the design more closely and making the recommended adjustments, your button will seamlessly integrate with the overall look of the project. Great work, and keep refining those details! 👍🎨🔍

File: src/components/BookForm.js [hunk 4]
```diff
@@ -24,18 +24,19 @@ const BookForm = () => {
...
-          <option value="romance">Romance</option>
-          <option value="action">Action</option>
-          <option value="technology">Technology</option>
+          <option value="Fantasy">Fantasy</option>
+          <option value="Science">Science</option>
+          <option value="Mystery">Mystery</option>
+          <option value="Romance">Romance</option>
+          <option value="Action">Action</option>
+          <option value="Technology">Technology</option>
         </select>
```
Comment (shahadat3669) [inline src/components/BookForm.js]: - [ ] 👏 Great work on adding the input fields, and they are quite close to the design! To make them align even more closely with the design, I recommend updating the border color to match the design as closely as possible. By making this adjustment, you'll bring the input fields in line with the project's overall visual consistency. Keep up the good work! 👍🎨📝

| Your | Design |
| --- | --- |
|  image | image|

File: src/components/Book.js [hunk 5]
```diff
@@ -1,21 +1,41 @@
...
-          <p className="book-title">{book.title}</p>
+          <h5 className="book-title">{book.title}</h5>
           <span className="book-author">{book.author}</span>
           <div className="btn-container">
             <button type="button" className="btn">Comment</button>
             <div className="line" />
-            <button type="button" className="btn" onClick={() => removeBook(book.item_id)}>Remove</button>
+            <button type="button" className="btn" onClick={() => removeBook(book.itemId)}>Remove</button>
             <div className="line" />
             <button type="button" className="btn">Edit</button>
```
Comment (shahadat3669) [inline src/components/Book.js]: - [ ] 👏 Good job on adding the vertical line between the buttons! However, to ensure it aligns perfectly with the design, I recommend updating the styling of the vertical line to match the design as closely as possible. Pay attention to details like color, thickness, and alignment to bring the element in line with the project's visual consistency. Keep up the great work! 👍🎨📏
- [ ] Second, please conder to use the proper font family mentioned in the design to match the styling.

| Your | Design |
| --- | --- |
|  image | image|

File: src/components/Book.js [hunk 6]
```diff
@@ -1,21 +1,41 @@
...
+        </div>
+        <div className="progress-bar">
+          <div className="progress-container">
+            <div style={{ width: 70, height: 70 }}>
+              <CircularProgressbar value={1} />
+            </div>
           </div>
+          <div className="text">
+            <span className="Percent-Complete">2%</span>
+            <span className="Percent-Complete1">Completed</span>
```
Comment (shahadat3669) [inline src/components/Book.js]: - [ ] 👏 Great work on adding the circular progress bar! However, to bring the text of this progressbar in line with the design, please consider updating the text's color, font styling, and other visual aspects to match the design as closely as possible. Attention to these details will ensure that your project's visual consistency is maintained. Keep up the good work! 👍🎨🔄

| Your | Design |
| --- | --- |
|  image| image|

File: src/components/Book.js [hunk 7]
```diff
@@ -1,21 +1,41 @@
...
+          </div>
+        </div>
+        <div className="Chapter-bar">
+          <span className="Current-Chapter">Current Chapter:</span>
+          <span className="Current-Lesson">Chapter 17</span>
+          <div className="Rectangle"><span className="update-progress">Update progress</span></div>
         </div>
```
Comment (shahadat3669) [inline src/components/Book.js]: - [ ] 👏 Great job adding the placeholder elements! However, there are some styling discrepancies in your implementation. As per the design, the "Current-Chapter" and "Current-Lesson" elements should be on separate lines, but currently, they are not following the design. Additionally, the button does not match the design. I recommend aligning your styling more closely with the design to improve the visual consistency of this section. Keep up the good work! 👍🎨📝

| Your | Design |
| --- | --- |
|  image| image|


- [ ] Additionally, there should be a vertical line between the "progress-bar" and "Chapter-bar," which is currently missing in your code. Please consider following the design and updating your code to include this line for a more accurate representation of the design.

| Your | Design |
| --- | --- |
|  image| image|

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @Obimbo07, 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ No repeating of the same rule
✅ No ID selectors for multiple elements
✅ No !important overusing


## Required Changes ♻️
👍 Great job on getting the project to work functionally! However, for a more polished look that aligns with the design, consider making the following adjustments:

- [ ] **Navbar**: In the design, the Navbar in this section should align along the vertical axis with the book card. To achieve this, you may need to work on the alignment and positioning of the elements in your code. Ensuring that everything lines up neatly will give your project a more professional and visually appealing appearance.
- [ ] **Navbar**: As per the design, there should be a border bottom for this navbar. Please consider following the design and adding the border as in the design.
- [ ] The Navitems for "books" and "Categories" do not have proper styling as in the design. Please consider following the design and updating your code to match the styling.
- [ ] **Navbar**: In the navbar, there should be a profile icon on the right side. Please consider following the design and adding the missing element.

| Your | Design |
| --- | --- |
|  image| image |

- [ ] As per the design, the app container should have some shadow, padding, border radius, and background color. I recommend following the design and updating your code to match the design's styling for a more polished appearance. Keep up the good work! 👍🎨👨‍💻

| Your | Design |
| --- | --- |
|  image | image|

By updating the alignment, your project will be in better harmony with the design, providing a more cohesive and polished user experience. Keep up the good work! 👍📐🎨


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 7 above)
Comment (Obimbo07) [reply src/components/Book.js]: Done implementing the changes.

File: src/components/Book.js [hunk 8]
```diff
@@ -1,20 +1,43 @@
...
 function Book({ book, removeBook }) {
   return (
     <div>
       <div className="book-section" key={book.item_id}>
         <div className="book-innerSection">
           <span className="book-category">{book.category}</span>
-          <p className="book-title">{book.title}</p>
+          <h5 className="book-title">{book.title}</h5>
           <span className="book-author">{book.author}</span>
           <div className="btn-container">
```
Comment (binhussen) [inline src/components/Book.js:15]: - Great Job for implementing previous reviiwer seggetion, I think you missed one thing the `book` and `category`  menus
so remove the `a` default style and fit the given design as the previous reviewer suggested to have a better design 👍

Comment (binhussen) [inline src/components/Book.js:15]: - Great job for improving the style based on previous reviwer seggetions., I think you missed one thing witch is to update the `nav items` `book` and `category`, so style them based on given design to have a better design as previous reviewer suggested 👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Obimbo07 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights

- Sticked given Design ✔️
- Well Stractured Code ✔️
- Used Git flow ✔️


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 8 above)
Comment (Obimbo07) [reply src/components/Book.js:15]: Done

**Review by OyePriscilla — APPROVED** [review]
Hi @Obimbo07 👏 👏 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Implemented nav_items according to the suggestion ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**